extras_require.json

```diff
@@ -10,5 +10,21 @@
     "sphinx",
     "sphinx_rtd_theme",
     "sphinx-gallery==0.5.0"
+  ],
+  "test": [
+    "pytest>=4.6",
+    "mypy",
+    "pytest-timeout",
+    "flaky",
+    "openml",
```

Comment:
I don't think we require openml here as a dependency.

Reply:
Sorry, removed.